delta/plugins/composite-views/src/main/scala/ch/epfl/bluebrain/nexus/delta/plugins/compositeviews/model/CompositeViewRejection.scala

```diff
@@ -70,44 +76,82 @@ object CompositeViewRejection {
     * @param provided the provided revision
     * @param current  the last known revision
     */
-  final case class RevisionNotFound(provided: Long, current: Long)
-      extends CompositeViewRejection(
-        s"Revision requested '$provided' not found, last known revision is '$current'."
-      )
+  final case class RevisionNotFound(provided: Long, current: Long) extends CompositeViewRejection {
+    val reason =
+      s"Revision requested '$provided' not found, last known revision is '$current'."
+  }
+
+  /**
+    * Rejection returned when too many sources are specified.
+    *
+    * @param provided the number of sources specified
+    * @param max      the maximum number of sources
+    */
+  final case class TooManySources(provided: Int, max: Int) extends CompositeViewRejection {
```

Comment:
extends `CompositeViewSourceRejection` ?

Reply:
I'm not sure if it should extend `CompositeViewSourceRejection`, since there is no error on the source, it's the error on composite view which has too many sources.

Reply:
It seems to me still a rejection related to the sources of the composite view.